projects/igniteui-angular/src/lib/grids/row.directive.ts

```diff
@@ -176,7 +176,7 @@ export class IgxRowDirective implements DoCheck, AfterViewInit, OnDestroy {
     public checkboxElement: IgxCheckboxComponent;
 
     @ViewChildren('cell')
-    protected _cells: QueryList<any>;
+    protected _cells: QueryList<CellType>;
```

Comment:
Because this is the actual row, I think we can have this as the actual cell list. This will remove the necessity to cast the cells to any in the tests.

Now the RowType, however can have a cells query list that is of type CellType. What do you think?

Reply:
Though this would mean that we have to either hide the rowList getter from grid.base or make it return RowType.

Reply:
It would be a breaking change if I do that. I'm just making the changes where nothing will break in terms of API/etc.,